support/support.rst

```diff
@@ -0,0 +1,67 @@
+.. class:: hide-rst-heading
+
+Support
+=======
+
+Please first read the docs, the existing issue tracker issues and mailing
+list posts -- a lot of stuff is already documented / explained / discussed /
+filed there.
+
+Issue Tracker
+-------------
+
+If you've found a bug or have a concrete feature request, please create a new
+ticket on the project's `issue tracker`_.
+
+For more general questions or discussions, IRC or mailing list are preferred.
+
+Chat (IRC)
+----------
+Join us on channel #borgbackup on chat.freenode.net.
+
+As usual on IRC, just ask or tell directly and then patiently wait for replies.
+Stay connected.
+
+You could use the following link (after connecting, you can change the random
+nickname you get by typing "/nick mydesirednickname"):
+
+http://webchat.freenode.net/?randomnick=1&channels=%23borgbackup&uio=MTY9dHJ1ZSY5PXRydWUa8
+
+.. _mailing_list:
```

Comment:
this is likely also referenced from the docs.

Reply:
I've addressed this in the borg PR.